spec/index.bs

```diff
@@ -634,20 +634,16 @@ Each such <dfn dfn>texture subresource</dfn> is uniquely identified by a
 
 We define <dfn dfn>subresource</dfn> to be either a whole buffer, or a [=texture subresource=].
 
-Some [=internal usage=]s are compatible with others.
-That means an implementation can consider multiple usages active at the same time.
-There is a limited set of <dfn dfn>allowed usage multiset</dfn>s, where each usage is compatible with the other:
-  - {[=internal usage/input=] | [=internal usage/constant=] | [=internal usage/storage-read=]} - in any combination
-  - {[=internal usage/storage=] | [=internal usage/storage-read=]} - in any combination
-  - {[=internal usage/storage-write=]} - used one or more times
-  - [=internal usage/attachment=] - singular usage
-  - {[=internal usage/attachment-read=] | [=internal usage/storage-read=] | [=internal usage/constant=]} - in any combination
-
-A multi-set differentiates between an `[=internal usage=]` included once versus multiple times.
-This difference is critical for [=internal usage/attachment=],
-which can't be used multiple times in the same render pass for the same [=texture subresource=].
-
-Enforcing that the usages are only combined into one of the [=allowed usage multiset=]s
+Some [=internal usage=]s are compatible with others. A [=subresource=] can be in a state
+that combines multiple usages together. We consider a list |U| to be 
```

Comment:
Should either wrap in an `algorithm` so the `|U|` var works, or remove the `| |`